consensus/pacemaker/module.go

```diff
@@ -60,6 +63,16 @@ type pacemaker struct {
 	logPrefix string
 }
 
+// Structure to handle delaying block preparation (reaping the block mempool)
+type latestPrepareRequest struct {
+	m              sync.Mutex
```

Comment:
Can you add comments to each of the types. The names alone don't make it clear to me how/why it sused.